lib/NameAudio.c

```diff
@@ -1,455 +1,455 @@
 	/* 397 */ { 32000, 0, NULL, "Inst_Marimba_Prim" },
 	/* 398 */ { 32000, 0, NULL, "Inst_Marimba_Hi" },
-	/* 399 */ { 32000, 0, NULL, "Inst_BellPad" },
+	/* 399 */ { 32000, 0, NULL, "Inst_Synth_Bell" },
 	/* 400 */ { 32000, 0, NULL, "Inst_Bouzouki_Prim" },
 	/* 401 */ { 32000, 0, NULL, "Inst_Bouzouki_Hi" },
-	/* 402 */ { 32000, 0, NULL, "Perc_EthnicSoft" },
-	/* 403 */ { 32000, 0, NULL, "Perc_EthnicHard" },
-	/* 404 */ { 32000, 0, NULL, "Perc_EthnicMute" },
+	/* 402 */ { 32000, 0, NULL, "Perc_Djembe_Open" },
+	/* 403 */ { 32000, 0, NULL, "Perc_Djembe_Slap" },
+	/* 404 */ { 32000, 0, NULL, "Perc_Djembe_Bass" },
 	/* 405 */ { 32000, 0, NULL, "Perc_Gong" },
-	/* 406 */ { 32000, 0, NULL, "Perc_WindChime" },
-	/* 407 */ { 32000, 0, NULL, "Perc_CongaOpen" },
-	/* 408 */ { 32000, 0, NULL, "Inst_WoodChime" },
-	/* 409 */ { 32000, 0, NULL, "Inst_VoicePad" },
-	/* 410 */ { 32000, 0, NULL, "Inst_ChantFlute_Low" },
-	/* 411 */ { 32000, 0, NULL, "Inst_ChantFlute_Prim" },
-	/* 412 */ { 32000, 0, NULL, "Inst_ChantFlute_Hi" },
-	/* 413 */ { 0, 0, &gSampleInfo[407], "Perc_CongaOpen" }, // Dublicate
-	/* 414 */ { 32000, 0, NULL, "Perc_CongaSoft" },
-	/* 415 */ { 32000, 0, NULL, "Perc_CongaMute" },
+	/* 406 */ { 32000, 0, NULL, "Perc_Bells" },
+	/* 407 */ { 32000, 0, NULL, "Perc_Conga_Mute" },
+	/* 408 */ { 32000, 0, NULL, "Inst_Bamboo" },
+	/* 409 */ { 32000, 0, NULL, "Inst_Pad_Voice" },
+	/* 410 */ { 32000, 0, NULL, "Inst_Chant_Flute_Low" },
+	/* 411 */ { 32000, 0, NULL, "Inst_Chant_Flute_Prim" },
+	/* 412 */ { 32000, 0, NULL, "Inst_Chant_Flute_Hi" },
+	/* 413 */ { 0, 0, &gSampleInfo[407], "Perc_Conga_Mute" }, // Dublicate
+	/* 414 */ { 32000, 0, NULL, "Perc_Conga_Open" },
+	/* 415 */ { 32000, 0, NULL, "Perc_Conga_Bass" },
 	/* 416 */ { 32000, 0, NULL, "Perc_Shaker" },
-	/* 417 */ { 32000, 0, NULL, "Perc_CowBell" },
-	/* 418 */ { 32000, 0, NULL, "Inst_JazzGuitar_Low" },
-	/* 419 */ { 32000, 0, NULL, "Inst_JazzGuitar_Prim" },
-	/* 420 */ { 16000, 0, NULL, "Inst_JazzGuitar_Hi" },
-	/* 421 */ { 32000, 0, NULL, "Inst_Fiddle" },
-	/* 422 */ { 22050, 0, NULL, "Perc_Bell" },
-	/* 423 */ { 16000, 0, NULL, "Perc_CuicaOpen" },
-	/* 424 */ { 16000, 0, NULL, "Perc_CuicaMute" },
-	/* 425 */ { 32000, 0, NULL, "Inst_Banjo_Prim" },
-	/* 426 */ { 32000, 0, NULL, "Inst_Banjo_Hi" },
-	/* 427 */ { 16000, 0, NULL, "Inst_Chant" },
-	/* 428 */ { 32000, 0, NULL, "Inst_SteelDrum_Prim" },
-	/* 429 */ { 32000, 0, NULL, "Inst_SteelDrum_Hi" },
-	/* 430 */ { 32000, 0, NULL, "Inst_FluteSynth" },
-	/* 431 */ { 32000, 0, NULL, "Inst_Wind2" },
-	/* 432 */ { 32000, 0, NULL, "Perc_CajonBass" },
-	/* 433 */ { 32000, 0, NULL, "Perc_CajonSnare" },
-	/* 434 */ { 32000, 0, NULL, "Inst_SynthPad" },
+	/* 417 */ { 32000, 0, NULL, "Perc_Cowbell" },
+	/* 418 */ { 32000, 0, NULL, "Inst_Guitar_Electric_Low" },
+	/* 419 */ { 32000, 0, NULL, "Inst_Guitar_Electric_Prim" },
+	/* 420 */ { 16000, 0, NULL, "Inst_Guitar_Electric_Hi" }, // weed
+	/* 421 */ { 32000, 0, NULL, "Inst_Violin" },
```

Comment:
I do not agree on this. Fiddle is more fitting for the style which this is played on.
```suggestion
	/* 421 */ { 32000, 0, NULL, "Inst_Fiddle" },
```

Reply:
Apparently there is no difference between a violin and a fiddle. It's just called a different name depending on the music, so yea, you're right.
![image](https://user-images.githubusercontent.com/99634136/155547687-d4bd60ba-cce7-4215-8b62-47ab454b9ebe.png)